mobile/bindings.go

```diff
@@ -0,0 +1,58 @@
+// +build ios android
+
+package lndmobile
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	flags "github.com/jessevdk/go-flags"
+	"github.com/lightningnetwork/lnd"
+)
+
+// Start starts lnd in a new goroutine.
```

Comment:
How is `lnd` meant to be torn down? Or is the assumption everything will die as soon as the parent process does? 

Is it possible for us to receive signals on the mobile systems as we do right now on the current base operating systems? Certain sub-systems like the tower client rely on being notified for a critical shutdown vs a regular one.